build.sbt

```diff
@@ -89,17 +90,9 @@ lazy val high = module("high")
 
 // connectors
 
-val awsSnykOverrides = Seq(
-  "commons-codec" % "commons-codec" % "1.15"
-)
-
-val nettyVersion = "4.1.94.Final"
-
 //Fixes https://security.snyk.io/vuln/SNYK-JAVA-IONETTY-5725787
 val nettySnykOverrides = Seq(
```

Comment:
are those overrides still needed?

Reply:
yes
it is not needed since i added `"software.amazon.awssdk" % "sqs/sns/kinesis" % Versions.AwsSdkV2` in each connector

thanks